specification/batch/Azure.Batch/models.tsp

```diff
@@ -495,7 +495,7 @@ union BatchNodeDeallocationOption {
   @doc("Allow currently running Tasks to complete. Schedule no new Tasks while waiting. Remove Compute Nodes when all Tasks have completed.")
   TaskCompletion: "taskcompletion",
 
-  @doc("Allow currently running Tasks to complete, then wait for all Task data retention periods to expire. Schedule no new Tasks while waiting. Remove Compute Nodes when all Task retention periods have expired.")
+  @doc("Deprecating. Please upload task data to azure storage in your task and use TaskCompletion instead.")
```

Comment:
I believe the right way to do this in TypeSpec would be to leave the doc string alone and add a [#deprecated directive](https://typespec.io/docs/language-basics/directives/#deprecated) to the enum value.

If we're already treating this as deprecated I think we should also say "Deprecated" instead of "Deprecating", which is ambiguous. Also, "Azure Storage" should be capitalized.

Reply:
Looks good!